frontend/views/containers/chatroom/MessageActionsMobile.vue

```diff
@@ -52,11 +52,6 @@ export default ({
     isActive () {
       return this.Menu.isActive
     }
-  },
-  watch: {
-    isActive (newVal) {
-      console.log('!@# isActive: ', newVal)
-    }
```

Comment:
Found out this was not noticed in this merged PR https://github.com/okTurtles/group-income/pull/2715. Removing it here.